### DISCUSSION TITLE
请求支持cloudreve列表

### DISCUSSION THREAD

**Discussion by Youwenqwq** [AlistGo/alist#499] [Ideas]
大部分cloudreve搭建成的私人网盘分享链接可以不登录下载文件，请问大佬能否支持一下这个网盘
官网地址https://cloudreve.org/

**Comment by fengguowudi** [comment]
可以通过webdav支援

**Reply by Youwenqwq** [reply]
可以尝试

**Reply by Youwenqwq** [reply]
若网盘管理员关闭了Webdav，会显示403Forbidden，还是得作者提供支援
